Production/govuk_ios/Resources/Constants.swift

```diff
@@ -3,5 +3,11 @@ import Foundation
 struct Constants {
     struct API {
         static let privacyPolicyUrl = "https://www.gov.uk/government/publications/govuk-app-privacy-notice-how-we-use-your-data"
+#if DEBUG
+        static let appConfigUrl = "https://app.integration.publishing.service.gov.uk"
+#else
+        static let appConfigUrl = "https://app.publishing.service.gov.uk"
```

Comment:
I believe that if we deploy to testflight, it will not be a debug build, and so testers will try and hit this endpoint, which is not currently live.

Reply:
This is correct, I would just leave integration in for now and we worry about other environments in a bit

Reply:
Done